boot/zephyr/sample.yaml

```diff
@@ -5,7 +5,38 @@ sample:
 tests:
   sample.bootloader.mcuboot:
     tags: bootloader_mcuboot
-    platform_allow:  nrf52840dk/nrf52840 frdm_k64f disco_l475_iot1
+    platform_allow:
+      - frdm_k22f
+      - frdm_k64f
+      - frdm_k82f
+      - frdm_ke17z
+      - frdm_ke17z512
+      - rddrone_fmuk66
+      - twr_ke18f
+      - twr_kv58f220m
+      - frdm_mcxn947/mcxn947/cpu0
+      - lpcxpresso55s06
+      - lpcxpresso55s16
+      - lpcxpresso55s28
+      - lpcxpresso55s36
+      - lpcxpresso55s69/lpc55s69/cpu0
+      - mimxrt1010_evk
+      - mimxrt1015_evk
+      - mimxrt1020_evk
+      - mimxrt1024_evk
+      - mimxrt1040_evk
+      - mimxrt1050_evk
+      - mimxrt1060_evk
+      - mimxrt1062_fmurt6
+      - mimxrt1064_evk
+      - mimxrt1160_evk/mimxrt1166/cm7
+      - mimxrt1170_evk/mimxrt1176/cm7
+      - vmu_rt1170/mimxrt1176/cm7
+      - mimxrt595_evk/mimxrt595s/cm33
+      - mimxrt685_evk/mimxrt685s/cm33
+      - rd_rw612_bga
+      - nrf52840dk/nrf52840
+      - disco_l475_iot1
```

Comment:
Since this is reworked... can we have these in alphabetical order?

Reply:
> > Since this is reworked... can we have these in alphabetical order?
> 
> I tried to have it per vendor, not alphabetical.

Yeah, so then you have primary by vendor, secondary by alphabet, or primary by vendor secondary by chaos.
Or just everything goes by alphabet. 

Anyway, up to you, but I am in favor of everything alphabetically. 

> 
> BTW: Just an idea. Can we add "mcuboot" to the board's .yaml "supported:" section and avoid adding boards to the allow list?

Haven't thought about that.

Reply:
> Yeah, so then you have primary by vendor, secondary by alphabet, or primary by vendor secondary by chaos.
> Or just everything goes by alphabet.

Not so easy;) 
By vendor, then by family (kinetis, RT, LPC...), so it may look like a bit chaotic but better for real searching.

>Haven't thought about that.

It's not convenient to update all these lists in different  tests/samples.

Reply:
> Yeah, so then you have primary by vendor, secondary by alphabet, or primary by vendor secondary by chaos.
> Or just everything goes by alphabet.

@de-nordic 
According your request, it has been updated by vendor and after by alphabet.

BTW:
CI is broken.

Reply:
> CI is broken.

Very strange.
According error logs:
- It reported error "Missing "Signed-off-by: Andrej Butok <Andrey.Butok@nxp.com>" in commit. I had "Signed-off-by: Andrej Butok <andrey.butok@nxp.com>"
- So I have changed the commit message to requested  "Signed-off-by: Andrej Butok <Andrey.Butok@nxp.com>"
- After that CI reports error "Missing "Signed-off-by: Andrej Butok <andrey.butok@nxp.com>".
It looped its requirements :)

Should I close the PR and try to create a new one?